src/common/pathAggregator.go

```diff
@@ -167,12 +167,17 @@ func buildPathTree(treeMap map[string]*Node, paths []string) {
 
 		rootPath := tokenizedPaths[0]
 
+		if rootPath == "/tmp" {
```

Comment:
Why are we specifically adding tmp for check here @achrefbensaad ?

Reply:
tmp files creates by applications under /tmp have random names and are not consistant between pod restarts, so we need to allow /tmp/**.
Adding tmp there is the simplest way to guarenty that behavior.